src/fonduer/learning/classifier.py

```diff
@@ -403,7 +403,7 @@ def predict(self, X, b=0.5, pos_label=1, return_probs=False):
         self.logger.info(f"Using positive label class {pos_label} with threshold {b}")
 
         Y_pred = np.array(
-            [pos_label if p[pos_label - 1] > b else 3 - pos_label for p in Y_prob]
+            [pos_label if p[pos_label] > b else pos_label - 1 for p in Y_prob]
```

Comment:
The new label starts from 0. So, in binary case the other case is `1 - pos_label`?

Reply:
Good catch! Resolved by 3ba6cf0048490e15161f3ae232735b812a7667eb.